src/LodeRunner.Data/Interfaces/ITestRunService.cs

```diff
@@ -14,7 +14,7 @@ namespace LodeRunner.Data.Interfaces
     /// <summary>
     /// TestRun Interface.
     /// </summary>
-    public interface ITestRunService
+    public interface ITestRunService : IBaseService<TestRun>
```

Comment:
My merged changes removed the redundant declarations for `Post()` and `Delete()`.  Additionally, I shortened the name for the one signature I left here.  I changed:
```csharp
Task<IEnumerable<TestRun>> GetAvailableTestRunsByClientIdAsync(string clientId);
```
To (including the entire interface declaration):
```csharp
    public interface ITestRunService : IBaseService<TestRun>
    {
        /// <summary>
        /// Gets all available TestRuns for the given client id.
        /// </summary>
        /// <param name="clientId">The client id.</param>
        /// <returns>List of TestRuns to run on client.</returns>
        Task<IEnumerable<TestRun>> GetNewTestRunsByClientId(string clientId);
    }
}
```
Please update this PR to maintain that change.